zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkEventThread.java

```diff
@@ -92,9 +92,7 @@ public String toString() {
       while (!isInterrupted()) {
         ZkEvent zkEvent = _events.take();
         int eventId = _eventId.incrementAndGet();
-        if (LOG.isDebugEnabled()) {
-          LOG.debug("Delivering event #" + eventId + " " + zkEvent);
-        }
+        LOG.debug("Delivering event #" + eventId + " " + zkEvent);
```

Comment:
Let's try to use interpolation, otherwise the string concatenation might be unnecessary.